lacework/resource_lacework_alert_profile.go

```diff
@@ -0,0 +1,224 @@
+package lacework
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/lacework/go-sdk/api"
+	"github.com/pkg/errors"
+)
+
+func resourceLaceworkAlertProfile() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkAlertProfileCreate,
+		Read:   resourceLaceworkAlertProfileRead,
+		Update: resourceLaceworkAlertProfileUpdate,
+		Delete: resourceLaceworkAlertProfileDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkAlertProfile,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:             schema.TypeString,
+				Required:         true,
+				Description:      "The name of the alert profile",
+				ValidateDiagFunc: StringDoesNotHavePrefix("LW_"),
+			},
+			"extends": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of existing alert profile from which this profile extends",
+			},
+			"alert": {
+				Type:        schema.TypeSet,
+				MinItems:    1,
+				Required:    true,
+				Description: "The list of alert templates",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:        schema.TypeString,
+							Description: "The name that policies can use to refer to this template when generating alerts",
+							Required:    true,
+						},
+						"event_name": {
+							Type:        schema.TypeString,
+							Description: "The name of the resulting alert",
```

Comment:
nit
```suggestion
							Description: "The name of the resulting alert event",
```

Reply:
I'm going to leave these as is, they're lifted from the api docs